scipy/optimize/tests/test_brent.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import division
+
+from numpy.testing import (assert_almost_equal, assert_array_less, assert_)
+
+from scipy.optimize._brent import find_minimum
+
+def test_strictly_convex():
```

Comment:
It would be good to test how it behaves (in terms of tolerance) when the minimum value is zero - relative versus absolute tolerance matters here, and what users want can differ. Also some cases where the minimum is at a large (e.g. 1/epsilon) position or has a large value (e.g. 1/sqrt(epsilon)).